cn/docs/tasks/run-application/run-stateless-application-deployment.md

```diff
@@ -0,0 +1,147 @@
+---
+title: 使用Deployment运行一个无状态应用
+---
+
+{% capture overview %}
+
+本文介绍通过Kubernetes Deployment对象如何去运行一个应用.
+
+{% endcapture %}
+
+
+{% capture 目标 %}
```

Comment:
The `capture` variable names need to be in English and follow the ones outlined in https://kubernetes.io/docs/home/contribute/page-templates/#task_template, i.e.:

* `overview`
* `prerequisites`
* `steps`
* `discussion`
* `whatsnext`

Reply:
ok ,thanks,I have updated it!